testing-proto/build.gradle

```diff
@@ -12,7 +12,6 @@ dependencies {
             project(':grpc-stub')
     compileOnly libraries.javax_annotation
     testImplementation libraries.truth
-    testRuntime libraries.javax_annotation
```

Comment:
Why is this removed. Does it work for all JDK versions?

Reply:
`testRuntime` is the old-style. It seemed weird to be runtime-only. It seemed like it should be `testImplementation`. So I looked at the code and couldn't find a reference to it, so I deleted it. The Java 11 build did fail, so I put it back, but as `testRuntimeOnly`.